mteb/benchmarks/get_benchmark.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+import difflib
+
+import mteb.benchmarks.benchmarks as benchmark_module
+from mteb.benchmarks.benchmarks import Benchmark
+
+BENCHMARK_REGISTRY = {
+    inst.name: inst
+    for nam, inst in benchmark_module.__dict__.items()
+    if isinstance(inst, Benchmark)
+}
+
+
+def get_benchmark(
+    benchmark_name: str,
+) -> Benchmark:
+    if benchmark_name not in BENCHMARK_REGISTRY:
```

Comment:
Very cool!

Reply:
We do the same for tasks in mteb.get_task. I misspelled so many times that I added it.